### PR TITLE
View more absences

### PR DESCRIPTION
**Pull request by JohnThirlaway1**
changed indentation and cleaned up code
connect to #16

### CODE REVIEW

File: app/components/MyCalendar.js [hunk 1]
```diff
@@ -214,82 +218,82 @@ var MyCalendar = React.createClass({
...
-          </span>
-          <span className='days-item other'>
-            <span>Other:</span>
-            <div>
-              {this.state.Other}
-            </div>
-          </span>
-          <span className='add-absence'>
-            <button onClick={() => this.showCreateAbsence()} className="add-absence-button abs-button-primary">CREATE ABSENCE<span className="glyphicon glyphicon-plus"></span></button>
+    <div id='myCalendar'>
```
Comment (TheGarrett) [inline app/components/MyCalendar.js]: Indentation

File: app/components/viewAbsence.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+var ViewAbsences = React.createClass({
+  hideAbsenceDetails: function () {
+    // Remove the create absence popup
+    document.getElementById('greyOut').classList.toggle('display-none', true);
+    document.getElementById('viewAbsence').classList.toggle('display-none', true);
+  },
+  getInitialState: function () {
+    return {
+        absencesDetails: [
```
Comment (TheGarrett) [inline app/components/viewAbsence.js]: indentation

File: app/styles/css/calendar.css [hunk 3]
```diff
@@ -555,3 +555,56 @@
...
+  padding: 5px 20px;
+  position: absolute;
+  right: 20px;
+}
+.Absent-title {
+  margin-bottom: 20px;
+  margin-top: 10px;
+}
+.display-none-absence {
+    display: none;
```
Comment (TheGarrett) [inline app/styles/css/calendar.css]: indentation

**Comment by adamk5550** [comment]
Don't merge until dashboard is merged

File: app/components/MyCalendar.js [hunk 4]
```diff
@@ -191,11 +192,14 @@ var MyCalendar = React.createClass({
     document.getElementById('greyOut').classList.toggle('display-none', false);
     document.getElementById('createAbsence').classList.toggle('display-none', false);
   },
-
   createNewAbsence: function (dateFrom, dateTo, reason) {
     this.refs.YearCalendar.newAbsence(dateFrom, dateTo, reason);
   },
-
+  showAbsenceDetails: function (absenceID) {
+    //this.refs.viewAbsence.viewNewAbsence(absenceID);
```
Comment (joerushbrooke) [inline app/components/MyCalendar.js]: Remove comment. Unnecessary

File: app/components/MyCalendar.js [hunk 5]
```diff
@@ -191,11 +192,13 @@ var MyCalendar = React.createClass({
     document.getElementById('greyOut').classList.toggle('display-none', false);
```
Comment (adamk5550) [inline app/components/MyCalendar.js]: duplicate code?

File: app/components/TeamChart.js [hunk 6]
```diff
@@ -293,15 +294,39 @@ var TeamChart = React.createClass({
...
+          to: eventItem.end,
+          type: eventItem.className,
+          duration: 0,
+          workingDays: 0,
+          approver: '',
+          notify: ''
+        }
+        self.props.showAbsence(eventObject);
+      }
+    })
```
Comment (TheGarrett) [inline app/components/TeamChart.js]: missing semi-colon

File: app/components/TeamChart.js [hunk 7]
```diff
@@ -293,15 +294,39 @@ var TeamChart = React.createClass({
...
 
     // Create a Timeline
     var timeline = new vis.Timeline(container, items, groups, options);
+
+    container.addEventListener('click', function (e) {
+      var props = timeline.getEventProperties(e);
+      var user = groups[props.group-1];
+      var eventItem = items.get(props.item);
+
+      if(props.item !== null) {
```
Comment (TheGarrett) [inline app/components/TeamChart.js]: space after the if

File: app/components/TeamChart.js [hunk 8]
```diff
@@ -293,15 +294,39 @@ var TeamChart = React.createClass({
       zoomMax: 5184000000,
       groupTemplate: function (group) {
         return '<img src="app/images/Profile.png" />' + group.content;
+      },
+      template: function (item) {
+        return '<span onclick='
```
Comment (TheGarrett) [inline app/components/TeamChart.js]: missing semi colon

File: app/components/TeamChart.js [hunk 9]
```diff
@@ -293,15 +294,39 @@ var TeamChart = React.createClass({
...
+        var eventObject = {
+          user: user.content,
+          from: eventItem.start,
+          to: eventItem.end,
+          type: eventItem.className,
+          duration: 0,
+          workingDays: 0,
+          approver: '',
+          notify: ''
+        }
```
Comment (TheGarrett) [inline app/components/TeamChart.js]: missing semi colon

File: app/components/TeamChart.js [hunk 10]
```diff
@@ -293,15 +294,39 @@ var TeamChart = React.createClass({
...
+        return '<span onclick='
       }
     };
 
     // Create a Timeline
     var timeline = new vis.Timeline(container, items, groups, options);
+
+    container.addEventListener('click', function (e) {
+      var props = timeline.getEventProperties(e);
+      var user = groups[props.group-1];
```
Comment (TheGarrett) [inline app/components/TeamChart.js]: needs to be spaced like this... [props.group - 1];

**Pull request merged**